draft-ietf-gnap-core-protocol.md

```diff
@@ -1294,22 +1300,22 @@ formats using a single proofing mechanism.
 [Continuation requests](#continue-request)
 MUST use the same key (or its most recent rotation) and proof method as the initial request.
 
-### Providing Displayable RC Information {#request-display}
+### Providing Displayable Client Instance Information {#request-display}
 
-If the RC has additional information to display to the RO
+If the client instance has additional information to display to the RO
 during any interactions at the AS, it MAY send that information in the
 "display" field. This field is a JSON object that declares information
 to present to the RO during any interactive sequences.
 
 
 name (string)
-: Display name of the RC software
+: Display name of the client instance or client software
```

Comment:
When would be appropriate to identify the instance vs the software here? I feel like that needs some clarification